src/main/java/org/mitre/synthea/world/concepts/Claim.java

```diff
@@ -332,4 +324,41 @@ public BigDecimal getTotalCoinsurancePaid() {
   public BigDecimal getTotalPatientCost() {
     return this.totals.getPatientCost();
   }
+
+  /**
+   * Returns whether this Claim was covered by Medicare as the primary payer.
+   * @return
+   */
+  public boolean coveredByMedicare() {
+    // TODO - Should there be a check here for Dual Eligble?
```

Comment:
Let me know how this should be handled.

Reply:
I don't understand why you are only looking for `MEDICARE` and not also `DUAL_ELIGIBLE` as in the commented out code? Is the concern about charging medicare for costs that would be covered by medicaid?

Reply:
This is related to the RIF exporters, for columns matching the pattern `*_PRMRY_PYR_PD_AMT`:
> The amount of a payment made on behalf of a Medicare beneficiary by a primary payer other than Medicare, that the provider is applying to covered Medicare charges for to the line item service on the non-institutional claim.

The idea is that we need to know if the primary payer is Medicare or not. A Dual-Eligible is being covered by Medicare as the primary payer.

So, yes, there should be a check for duals as well.